### PR TITLE
AC#1033/fix-in-translation-endpoint

### PR DESCRIPTION
**Pull request by zingmane**
# Submit a pull request

## Please make sure the following is true

- [x] I gave the PR a meaningful name
- [x] I checked that the correct target branch is selected
- [x] I rebased the branch on the target branch and it can be merged
- [x] I ran the linter and it did pass
- [x] I checked for unused code / dead code / debug code
- [x] I checked that variables/functions have meaningful names
- [x] I checked that the behaviour is as the documentation/task describes and I tested it
- [x] I updated the docs / specifications if possible
- [x] I could explain all that code when someone wakes me up at 3am
- [x] I checked that the code considers failures and not just the happy path
- [x] There are no new dependencies OR I listed them and explained them below
- [x] PR introduces no breaking changes OR I listed them and described them below
- [x] I added/updated tests for new/modified unit-testable functions/helpers
- [x] I ran the tests and they did pass

## Other information/comments (e.g. reasons why points are not checked from above)

In dem Zuge hab ich auch das logging verbessert und im backend wird nicht mehr gelogged, wenn man eine Zeile per `retrieveForeignRows` nicht sehen darf. Durch die Rekursion und bei vielen Ausgangszeilen würde das nämlich die logs einfach nur voll spammen. 

Das Dashboard hat mit der Änderung kein Problem mehr und rendert auch, wenn es nicht eine multicolumn Spalte gibt. Die Seite ist dann zwar witzlos, aber es wird auch nichts falsches angezeigt:

image

### CODE REVIEW

File: src/main/scala/com/campudus/tableaux/controller/TableauxController.scala [hunk 1]
```diff
@@ -232,7 +232,12 @@ class TableauxController(
           })
           .map(_._2)
 
-        (langtag, mergedTranslationStatusForLangtag.sum / mergedTranslationStatusForLangtag.size)
+        val calculatedLangtagStatus = mergedTranslationStatusForLangtag.size match {
```
Comment (hermann-p) [inline src/main/scala/com/campudus/tableaux/controller/TableauxController.scala:235]: Ob es für zwei mögliche Fälle, die nicht auf den Wert eines Datentypen zugreifen müssen, guter Stil ist, Pattern-Matching zu machen sei mal dahingestellt. Aber dass wir das sowohl beim einbauen als auch beim Review übersehen konnten - shame on us :grin:

**Review by hermann-p — APPROVED** [review]
LGTM. Ob dieser Edge-Case noch einen Test haben sollte?

**Review by Zwergal — APPROVED** [review]
lgtm

File: src/main/scala/com/campudus/tableaux/controller/TableauxController.scala (hunk 1 above)
Comment (zingmane) [reply src/main/scala/com/campudus/tableaux/controller/TableauxController.scala:235]: Hatte auch erst einen if, aber pattern matching ist einfach ausdrucksstärker, deswegen hab ichs geändert
